### PR TITLE
{tools}[GCCcore/8.3.0] Sumo v1.3.1 w/ Python 3.7.4

### PR DESCRIPTION
**Pull request by ekieffer**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by ekieffer** [comment]
Test report by @ULHPC
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
iris-002 - Linux centos linux 7.7.1908, x86_64, Intel(R) Xeon(R) CPU E5-2680 v4 @ 2.40GHz, Python 3.6.8
See https://gist.github.com/da2f937d5e5245f42565d404a4f4d659 for a full test report.

File: easybuild/easyconfigs/s/Sumo/Sumo-1.3.1-GCCcore-8.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,34 @@
+easyblock = 'CMakeMake'
+
+name = 'Sumo'
+version = '1.3.1'
```
Comment (smoors) [inline easybuild/easyconfigs/s/Sumo/Sumo-1.3.1-GCCcore-8.3.0.eb:4]: @ekieffer please a Python versionsuffix:
```suggestion
version = '1.3.1'
versionsuffix = '-Python-%(pyver)s'
```

File: easybuild/easyconfigs/s/Sumo/Sumo-1.3.1-GCCcore-8.3.0.eb [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+homepage = "https://sumo.sourceforge.net/"
+description = """Sumo is an open source, highly portable, microscopic and continuous traffic simulation package 
+ designed to handle large road networks."""
+
+toolchain = {'name': 'GCCcore', 'version': '8.3.0'}
+local_download_suffix = 'download.php?submit=Download&version=v%(version_major_minor)s&type=source&os=all&downloadFile='
```
Comment (smoors) [inline easybuild/easyconfigs/s/Sumo/Sumo-1.3.1-GCCcore-8.3.0.eb:11]: @ekieffer this local parameter is not used anywhere, so can we remove it?

Comment (ekieffer) [reply easybuild/easyconfigs/s/Sumo/Sumo-1.3.1-GCCcore-8.3.0.eb:12]: Ah yes, sure ! forgot to remove it. thx

**Comment by smoors** [comment]
@ekieffer why did you close this?

**Pull request closed without merging**